mappings/net/minecraft/screen/ScreenHandler.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/class_71 net/minecraft/screen/ScreenHandler
+	FIELD field_2731 revision S
+	FIELD field_2733 trackedStacks Ljava/util/List;
+	FIELD field_2734 slots Ljava/util/List;
+	FIELD field_2735 syncId I
+	FIELD field_2736 listeners Ljava/util/List;
+	METHOD method_2076 addListener (Lnet/minecraft/class_633;)V
+		ARG 1 listener
+	METHOD method_2078 onSlotClick (IIZLnet/minecraft/class_54;)Lnet/minecraft/class_31;
+		ARG 1 slotIndex
+		ARG 2 button
+		ARG 3 shift
+		ARG 4 player
+	METHOD method_2079 addSlot (Lnet/minecraft/class_133;)V
+		ARG 1 slot
+	METHOD method_2084 getSlot (I)Lnet/minecraft/class_133;
+		ARG 1 slotIndex
+	METHOD method_2086 getStackInSlot (I)Lnet/minecraft/class_31;
+		ARG 1 slotIndex
+	METHOD method_2087 onClosed (Lnet/minecraft/class_54;)V
+		ARG 1 player
+	METHOD method_2090 nextRevision (Lnet/minecraft/class_136;)S
```

Comment:
Same issue as comment above, what is the "revision" about?

Reply:
Refer to yarn - https://github.com/FabricMC/yarn/blob/bd3e21e1cd577dc1b0f4ec41cc88d8933e78c7e1/mappings/net/minecraft/screen/ScreenHandler.mapping#L137